src/uu/uname/src/uname.rs

```diff
@@ -61,33 +64,33 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         || hardware_platform);
 
     if kernel_name || all || none {
-        output.push_str(&uname.sysname());
+        output.push_str(&uname.sysname().to_string_lossy());
         output.push(' ');
     }
 
     if nodename || all {
         // maint: [2023-01-14; rivy] remove `.trim_end_matches('\0')` when platform-info nodename-NUL bug is fixed (see GH:uutils/platform-info/issues/32)
-        output.push_str(uname.nodename().trim_end_matches('\0'));
+        output.push_str(uname.nodename().to_string_lossy().trim_end_matches('\0'));
```

Comment:
I think `trim_end_matches` and the comment can be removed because https://github.com/uutils/platform-info/issues/32 has been fixed.